### PR TITLE
Release v0.16.0 changes

### PR DESCRIPTION
**Pull request by reinecke**
* drop .dev1 suffix
* update CONTRIBUTORS.md
* Added note to the README.md about OpenTimelineIO-Plugins
* c++ code formatted

This is the release of items from [Milestone 18 (Public Beta 16)](https://github.com/AcademySoftwareFoundation/OpenTimelineIO/milestone/18)

### CODE REVIEW

**Review by meshula — APPROVED** [review]
whitespace changes aside LGTM

**Comment by meshula** [comment]
(I'm not objecting to landing the ws changes, just noting that I checked to see if anything changed within the ws and didn't spot such, a separate ws PR would've required less brain juice ;) )

**Comment by jminor** [comment]
@reinecke you mentioned "check the README.md for any updates" in the description, but I don't see any README changes in this PR. Did I misunderstand something?

**Comment by reinecke** [comment]
@jminor sorry - that was a copy/paste error in my change summary. No changes intended to README.

**Comment by darbyjohnston** [comment]
It would also be nice to relax the 80 column limit, I'm not sure some of those formatting changes are an improvement.

**Comment by ssteinbach** [comment]
It might make more sense to have a deliberate command that executes the C++ format.  The reason we landed on this pattern was that we can be permissive as people work and conform the style together in one shot.  Typically, I would run the C++ format as a separate command and poke through the code, eg: https://github.com/AcademySoftwareFoundation/OpenTimelineIO/commit/0aa7765b6f620af9e456bd93509eff473c527e3a

that would let the CI run and folks look at stuff before landing it.

**Comment by ssteinbach** [comment]
oh yeah - and the checklist suggests reading through the readme to see if anything changed.  We might want to add something to the readme for this release noting that its the last release with all the adapters together?  I'm not sure off the top of my head of anything else that should be added there.

**Comment by reinecke** [comment]
@ssteinbach @jminor I made a quick update to the readme noting people should start using `OpenTimelineIO-Plugins` to get the full compliment of adapters.

File: README.md [hunk 1]
```diff
@@ -67,6 +67,12 @@ To provide interoperability with other file formats or applications lacking a
 native integration, the opentimelineio community has built a number of python
 adapter plugins. This includes Final Cut Pro XML, AAF, CMX 3600 EDL, and more.
 
+**Note: v0.16 will be the last release of the
+[OpenTimelineIO PyPI package](https://pypi.org/project/OpenTimelineIO/)
+to include the full set of adapter plugins. If you rely on these please use the
+[OpenTimelineIO-Plugins PyPI Package](https://pypi.org/project/OpenTimelineIO-Plugins/)
+to continue to receive them. Each OpenTimelineIO release will have a matching OpenTimelineIO-Plugins release.**
+
```
Comment (ssteinbach) [inline README.md:75]: I think rearranging this a bit more in the positive might help:

```suggestion
**Note: for releases after v0.16, users of the [OpenTimelineIO PyPI package](https://pypi.org/project/OpenTimelineIO/) will also need to install a separate [OpenTimelineIO-Plugins PyPI Package](https://pypi.org/project/OpenTimelineIO-Plugins/) to include the full set of adapter plugins.  Each OpenTimelineIO release will have a matching OpenTimelineIO-Plugins release.**

For more information: https://github.com/AcademySoftwareFoundation/OpenTimelineIO/issues/1386
```

Comment (reinecke) [reply README.md:75]: I like the change overall, I just worry that the wording might imply you need to install both (which you don't).

What if we do this:

```
**Note: for releases after v0.16, the [OpenTimelineIO PyPI package](https://pypi.org/project/OpenTimelineIO/) will only include the core libraries and file formats. Users that need the full set of adapter plugins should use the [OpenTimelineIO-Plugins PyPI Package](https://pypi.org/project/OpenTimelineIO-Plugins/). Each OpenTimelineIO release will have a matching OpenTimelineIO-Plugins release.**

For more information: https://github.com/AcademySoftwareFoundation/OpenTimelineIO/issues/1386
```

**Comment by Kanyiri234297** [comment]
552888

**Pull request merged**